src/main/java/com/epam/training/student_anton_lapushenko/WebDriver/Task3/page/GoogleCloudCalculatorPage.java

```diff
@@ -0,0 +1,127 @@
+package com.epam.training.student_anton_lapushenko.WebDriver.Task3.page;
+
+import com.epam.training.student_anton_lapushenko.WebDriver.Task3.enums.GPUModels;
+import com.epam.training.student_anton_lapushenko.WebDriver.Task3.enums.NumberOfGPUs;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+import java.util.List;
+
+public class GoogleCloudCalculatorPage {
+    WebDriver driver;
+
+    public GoogleCloudCalculatorPage(WebDriver driver) {
+        this.driver = driver;
+    }
+
+    private final By addToEstimateButtonBy = By.className("UywwFc-vQzf8d");
+    private final By computerEngineBy = By.xpath("(//*[@class='VobRQb'])[1]");
+
+    public void addComputerEngineToEstimate(){
+        WebElement estimateButton = driver.findElement(addToEstimateButtonBy);
+        estimateButton.click();
+//        new WebDriverWait(driver, Duration.ofSeconds(10))
+//                .until(ExpectedConditions
+//                        .elementToBeClickable(computerEngineBy)).click();
+        WebElement computerEngineButton = driver.findElement(computerEngineBy);
+        computerEngineButton.click();
+    }
+
+    private final By countInstancesBy = By.xpath("(//*[@class='qdOxv-fmcmS-yrriRe qdOxv-fmcmS-yrriRe-OWXEXe-INsAgc qdOxv-fmcmS-yrriRe-OWXEXe-di8rgd-V67aGc qdOxv-fmcmS-yrriRe-OWXEXe-V67aGc-NLUYnc'])[1]");
```

Comment:
try to avoid using of generated class names in your locators. These names can be dynamic and could change that would produce a lot of auto issues in future